### ISSUE TITLE
Find Class... location

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#554]
Currently, menu option ``Find Class...`` is in two places - on the console's ``Browse`` menu and on the browser's ``Jadeite`` menu.  (There is no ``Browse`` menu in the projects menu bar)

@dalehenrich suggests we make it consistent which presumably means putting it on the ``Jadeite`` menu in the console. 

Need @LisaAlmarode opinion - Should ``Find Class...`` be moved to the ``Jadeite`` menu on the console's menu bar?

**Comment by dalehenrich** [comment]
You incorrectly presume that I want the item on the Jadeite menu ... My personal opinion is that the set of menus presented "should be the same" for "all of the windows" ... if that were true, then I would want the `Find class` menu item under the `Class` menu ... you are free to ignore my opinions:)

**Comment by LisaAlmarode** [comment]
No, I don't like the find class on the jadeite menu.  On the console, I rather want the Jadeite menu item for system wide settings (see my other request to move the display etc. to the edit menu).  The PB Jadeite menu item "New Project Browser" is the same as the Console Browse "Browse Projects", so the inconsitency is not limited to find class. 

While I also believe that the menus should be the same; but my conclusion is that the PB should have a "Browse" menu (I am not aware of a Class menu nor is Find Class under the PB Classes menu item... so I don't really understand the rest of Dale's comment)

As I think I've mentioned, I don't approve of general commands (not specific to the selected method, like browse implementors...) being on the methods menu.  A Browse menu on the PB would make me happy in several ways.

**Comment by dalehenrich** [comment]
> (I am not aware of a Class menu nor is Find Class under the PB Classes menu item... so I don't really understand the rest of Dale's comment)

In our current universe we have a `Browse menu` on the Console and a set of `Projects`, `Packages`, `Classes`, and `Methods` menus on the Project Browser ... The Console `Browser` menu is a partial collection of some menu items "cherry-picked" from the set of Project Browser menus. 

I **expect** to find `Find class` on the Project Browser `Classes` menu, but right now it is on the `Jadeite` menu ...

My mental picture  of the Project Browser is that each of the Named menus on the window menu bar "should duplicate" the menu available from the Pane from the same name ... this is not reality.

There are times when I have to remember to go to the menu with the same name as the pane that I am in and there are times when I have to activate the pane itself (be careful if you want to preserve current selections since the act of navigating/activating) can cause you to lose your current selection ... There are some menu items on the menu bar that act on the selected item, but the menus are not identical ... and I think that they "should be."

Which now brings us to the Console... The Console has a `Jadeite` menu that is completely different from the `Jadeite` menu on the Project Browser ... The Console has a `Browser` menu includes a subset of menu items extracted from the Project Browser menus ... In my opinion, I think that all of the menu items on the Project Browser that do not operate on a selection in the corresponding pane should be available on the Console ... and in my opinion if the same set of menus were available on the Console (`Projects`, `Packages`, `Classes`, and `Methods`) then I would **not** have to search for menu items (as I do now) as the set of menu items would be "identical" --- in the cases where a selection is required in the Project Brower, the Console variant could prompt for the name of the item or display a list of items to select from .. or use pattren matching (ala find class) where the user can pre-fileter the list ...

when it comes to `Find` ... I think there should be a `Find Project` on the Project menu, a `Find Package` on the Package menu and a `Find Class` menu on the Class menu ... etc. and so on ...

My opinions are based on the principle that it shouldn't matter which window I am in, I should be able to navigate to the place I want to be without first having to remember which window I need to open in order to find the menu that I need to use to do the thing I want to do --- in other words no "menu searching" ...

If this still isn't clear, then perhaps we should just sit down and I can explain "my opinion" in person ...

**Comment by LisaAlmarode** [comment]
So I think you mean, the Find class... is wrong in both places; it's not only that you want them in the same place...  you also want them the place they would be intuitive (it's a different mental model than I use for menu items, which is more of applies-to-selected, so not really intuitive for me, but a case can be made).

I think an overhaul is needed whatever model we have, but I hesitate to try to get this into Nov.

**Comment by dalehenrich** [comment]
Agree 100%- I never thought this would be a target for November ... what we have now is better than what it was before ... but I'm putting in my $0.02 now ... i repeat that these are my opinions and they can be ignored:)
